<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="0" hostname="EN510723" name="samsara.TestRegistratingNewAccount" tests="7" failures="1" timestamp="2020-02-20T11:29:45 CET" time="174.363" errors="0">
  <testcase name="testLoginPositiveCases" time="25.051" classname="samsara.TestRegistratingNewAccount"/>
  <system-out/>
  <testcase name="backToLoginPageViaSamsaraLinkOnCreateAccount" time="23.137" classname="samsara.TestRegistratingNewAccount">
    <failure type="java.lang.AssertionError" message="expected [Register new account.] but found [Resister new account.]">
      <![CDATA[java.lang.AssertionError: expected [Register new account.] but found [Resister new account.]
at org.testng.Assert.fail(Assert.java:97)
at org.testng.Assert.assertEqualsImpl(Assert.java:136)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:575)
at org.testng.Assert.assertEquals(Assert.java:585)
at samsara.TestRegistratingNewAccount.backToLoginPageViaSamsaraLinkOnCreateAccount(TestRegistratingNewAccount.java:64)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:134)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:597)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:816)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(Unknown Source)
at org.testng.TestRunner.privateRun(TestRunner.java:766)
at org.testng.TestRunner.run(TestRunner.java:587)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1187)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1109)
at org.testng.TestNG.runSuites(TestNG.java:1039)
at org.testng.TestNG.run(TestNG.java:1007)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- backToLoginPageViaSamsaraLinkOnCreateAccount -->
  <system-out/>
  <testcase name="backOnLoginPageViaLoginButtonFromAreateAccount" time="25.811" classname="samsara.TestRegistratingNewAccount"/>
  <system-out/>
  <testcase name="testLoginCodeWarningMessage" time="25.428" classname="samsara.TestRegistratingNewAccount"/>
  <system-out/>
  <testcase name="testLoginCodeWarningMessage" time="25.306" classname="samsara.TestRegistratingNewAccount"/>
  <system-out/>
  <testcase name="testLoginCodeWarningMessage" time="24.690" classname="samsara.TestRegistratingNewAccount"/>
  <system-out/>
  <testcase name="testLoginPositiveCases" time="24.940" classname="samsara.TestRegistratingNewAccount"/>
  <system-out/>
</testsuite> <!-- samsara.TestRegistratingNewAccount -->
